config/dev.env.js

```diff
@@ -4,5 +4,7 @@ const merge = require('webpack-merge')
 const prodEnv = require('./prod.env')
 
 module.exports = merge(prodEnv, {
-  NODE_ENV: '"development"'
+  NODE_ENV: '"development"',
```

Comment:
how come quotes are like this? 
also, are these supposed to say development? when it's merging prodEnv?

Reply:
Yeah good catch, will fix that. And I'm honestly not sure why the variables have to be written that way for a string, but that's the way it's done by default in the template so I assumed it was correct

Reply:
Okay apparently this is the dev env, and it just overwrites values from the prod one. I assumed as I think you did that this was the prod file